contour/src/main/kotlin/com/squareup/contour/ContourLayout.kt

```diff
@@ -157,7 +158,9 @@ open class ContourLayout(
   private val density: Float = context.resources.displayMetrics.density
   private val widthConfig = SizeConfig()
   private val heightConfig = SizeConfig()
-  private val geometry = ParentGeometry(widthConfig, heightConfig)
+  private val geometry = ParentGeometry(widthConfig, heightConfig) {
```

Comment:
nit: can we name the `paddingProvider` argument here? It's difficult to read what's being passed in trailing lambdas otherwise.